src/main/kotlin/com/vanniktech/maven/publish/MavenPublishPluginExtension.kt

```diff
@@ -55,4 +57,27 @@ open class MavenPublishPluginExtension(project: Project) {
   var repositoryPassword: String?
     get() = defaultTarget.repositoryPassword
     set(value) { defaultTarget.repositoryPassword = value }
+
+  /**
+   * Allows to add additional [MavenPublishTargets][MavenPublishTarget] to publish to multiple repositories.
+   * @since 0.7.0
+   */
+  val targets: NamedDomainObjectContainer<MavenPublishTarget> =
+    project.container(MavenPublishTarget::class.java) { MavenPublishTarget(it, "") }.apply {
```

Comment:
why do we need to specify the second parameter of `MavenPublishTarget`?

Reply:
I didn't want to make `releaseRepositoryUrl` nullable so it's needs something when initializing it. Nullable and then throwing later during configuration is probably better though.

Reply:
fixed d519b3264d996175081532d924712d634e79fb54